sql_scripts/2_states.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE IF NOT EXISTS states
+(
+    id SMALLSERIAL PRIMARY KEY,
+    name VARCHAR(20) NOT NULL
+);
+
+
+INSERT INTO states(name) VALUES ('Alaska');
```

Comment:
All inserts should be with ON CONFLICT action, cause we want to restrict inserts of duplicate values. Add id during INSERT statament. For example: INSERT INTO beers.states(id, name) VALUES (1, 'Alabama') ON CONFLICT DO NOTHING. Also don't forget to declare schema during each insert